### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by Simpleshaikh1**
I added styling to look like the design

### CODE REVIEW

File: src/components/BookList.jsx [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
+          <>
+             {arrOfBooks.map((book) => (
+            <Book key={book.id} book={book} />
+              ))}
+              
+              <CreateBook />
+          </>
+        )
+      }
+
```
Comment (brainconnect93) [inline src/components/BookList.jsx]: - [ ] Great effort working on the styling of the bookstore project, But currently I could not see the books added and the form input to add the book I would kindly suggest you work on it and be sure the books rendered on the page in order to be able to see the design you have implemented.

Below is the video recording of my claim.

[screen-capture.webm](https://github.com/Simpleshaikh1/Book-Store/assets/78634841/2495942b-a520-490c-9303-c13d3cf2d586)

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Simpleshaikh1, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Styling of the bookstore CMS project ✔️
- Zeplin Design template used for the design ✔️
- There is a good use of Git flow ✔️

### Changes Required

- [ ] I would kindly suggest you work on the MIT license link as currently the link is broken and throwing error 404

MIT License 👇 

mit


- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookList.jsx (hunk 1 above)
Comment (Simpleshaikh1) [reply src/components/BookList.jsx]: Thank you sir.

File: src/App.css [hunk 2]
```diff
@@ -17,27 +17,29 @@
...
-  gap: 30%;
-  border: 1px solid blue;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 4rem;
+  width: 90%;
+  padding: 1%;
+  margin-bottom: 1rem;
+  background-color: white;
+  border: 1px solid rgb(221, 229, 224);
```
Comment (bobb-Rob) [inline src/App.css:27]: - [ ] Well done on this project. But if you look closely at the Zeplin design, you will see a huge disparity with your project style implementation. See below the list of things you should fix 👇🏾 

    - Use the correct colors where necessary. inspect the Zeplin design to see,
    - Use the correct font family and font size as used in the Zeplin
    - Ensure unnecessary borders are removed from buttons.
    - The progress bar has a percentage figure beside it and it is not like you have done. compare them. You can use an npm package. 
    - The chapter text is missing 

You will more issues pointed out in the image below.

## Your design
image

## Zeplin design
image

Please work on the styling some more. I have not touched all the issues but you should ensure your project resembles the Zeplin design as much as possible.

**Review by bobb-Rob — COMMENTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @Simpleshaikh1, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**